bundledata.go

```diff
@@ -113,11 +117,11 @@ type ApplicationSpec struct {
 	// For a subordinate application, this actually represents
```

Comment:
This comment needs updating.  Before the prior work (2015) to make NumUnits not a pointer it was:

// For a subordinate service, this must be nil;
// otherwise a nil value represents the default
// of a single unit.